app/permissions/decidim/time_tracker/admin/permissions.rb

```diff
@@ -25,11 +25,20 @@ def allowed_task_action?
         end
 
         def allowed_questionnaire_action?
-          return unless permission_action.subject == :questionnaire
+          return unless permission_action.subject.in? [:questionnaire, :questionnaire_answers]
 
-          case permission_action.action
-          when :update, :export_answers
-            permission_action.allow!
+          if permission_action.subject == :questionnaire
+            case permission_action.action
+            when :export_answers, :update
+              permission_action.allow!
+            when :preview
+              permission_action.disallow!
```

Comment:
This doesn't work because `participatory_process/permissions.rb` also validates the `action` `preview` without taking into account the subject.
I think the only solution for now is to allow previews (doing it)